src/components/ArticleListView.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ArticlePreview from 'components/ArticlePreview';
+
+export default class ArticleListView extends React.Component {
```

Comment:
ArticleList is a better name. The View is a little redundant from the .jsx and from the fact this is in components directory.